scripts/polkadotjs_endpoints_to_preconfigured.py

```diff
@@ -255,8 +258,8 @@ def main():
     for endpoint in Endpoints:
         get_ts_file(endpoint.value, ts_file_path)
         polkadotjs_data = ts_constant_to_json(ts_file_path)
-        create_json_files(polkadotjs_data, CHAINS_FILE_PATH_DEV)
-        create_json_files(polkadotjs_data, CHAINS_FILE_PATH_PROD)
+        create_json_files(polkadotjs_data, CHAINS_FILE_PATH_DEV, endpoint.name)
+        # create_json_files(polkadotjs_data, CHAINS_FILE_PATH_PROD, endpoint.name)
```

Comment:
it won't update the prod files, is that intended?

Reply:
nice catch, forgot to remove after test